File: Swift-C++-IR-Comparison/test/stringsAndCharacters/Swift/Interpolation.swift
// String interpolation is a way to construct a new String value from
//  a mix of constants, variables, literals, and expressions by including
//  their values inside a string literal. You can use string interpolation
//  in both single-line and multiline string literals. Each item that you
//  insert into the string literal is wrapped in a pair of parentheses,
//  prefixed by a backslash (\):

@main
struct MyMain {

    static func main() {
        let multiplier = 3
        let message = "\(multiplier) times 2.5 is \(Double(multiplier) * 2.5)"
        // message is "3 times 2.5 is 7.5"
    }
}

File: Swift-C++-Testsuite/AutoTranspiledTests/try-0/Interpolation.cpp
 
#include <iostream>
#include <string>

int main()
{
    int multiplier = 3;
    std::string message = std::to_string(multiplier) + " times 2.5 is " + std::to_string(multiplier * 2.5);
    
     // message is "3 times 2.5 is 7.5"
    std::cout << message << std::endl;
    return 0;
}